tests/integration_tests/test_timeseries_feature.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+import torch
+
+from ludwig.features.timeseries_feature import TimeseriesInputFeature
+from tests.integration_tests.utils import timeseries_feature
+
+BATCH_SIZE = 2
+SEQ_SIZE = 10
+DEFAULT_FC_SIZE = 256
```

Comment:
We can make this smaller for speed purposes. Maybe 4?

Reply:
done.